isic/core/services/collection/doi.py

```diff
@@ -84,27 +96,54 @@ def _datacite_create_doi(doi: dict) -> None:
         timeout=5,
         json=doi,
     )
-    # TODO: check for already exists
-    r.raise_for_status()
+
+    try:
+        r.raise_for_status()
+    except HTTPError:
+        logger.exception(f"DOI draft creation failed: {r.json()}")
```

Comment:
I don't think the text of the error message was captured on Sentry or anywhere else before. It seems useful to capture it because the messages are quite helpful.